clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/tests/WgsUbamTests.scala

```diff
@@ -32,6 +24,24 @@ import java.util.UUID
 /** Tests of Clio's wgs-ubam functionality. */
 trait WgsUbamTests { self: BaseIntegrationSpec =>
 
+  def runUpsert(key: TransferWgsUbamV1Key,
+                metadata: TransferWgsUbamV1Metadata): Future[UUID] = {
+    val tmpMetadata = writeTmpJson(metadata)
+    runClient(
+      WgsUbamTests.upsert,
```

Comment:
This client command has a constant, seems like the others should too. But putting the constant in test code doesn't seem right, is it possible to put it in the code that actually determines what this value is in production?

Reply:
The actual value is auto-magically determined by caseapp. It's possible to change the value with an annotation, but from [this issue](https://github.com/alexarchambault/case-app/issues/6) it sounds like the macros behind the annotations can only deal with literal strings.

I'll play with the annotations to double-check the issue (it's a few years old), and also check if we can get the derived class names from the implicits that drive the whole thing. Worst case, I'll move the magic string definitions into the client with a big warning comment.